tests/_fixture/CheckoutPage.ts

```diff
@@ -0,0 +1,115 @@
+import { expect, Page } from '@playwright/test';
+import { Polly } from '@pollyjs/core';
+
+export default class CheckoutPage {
+    readonly page: Page;
+    readonly isReplay: boolean;
+    polly: Polly | undefined;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.isReplay = process.env.MODE?.toLowerCase() === 'replay';
+    }
+
+    removeSensitiveHeaders(): void {
+        // hack for BC dev store's root certificate issue during recording HAR
+        // https://stackoverflow.com/questions/31673587/error-unable-to-verify-the-first-certificate-in-nodejs
+        process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+        this.polly?.server.any().on('beforePersist', (_, recording) => {
+            const processHeaders = (headers: []): {} => {
+                const dummyData = '*';
+                const sensitiveHeaderNames = [
+                    'authorization',
+                    'cookie',
+                    'set-cookie',
+                    'token',
+                    'x-session-hash',
+                    'x-xsrf-token',
+                ];
+
+                return headers.map((header: { name: string; value: string }) => {
+                    if (sensitiveHeaderNames.includes(header.name)) {
+                        return { ...header, value: dummyData };
+                    }
+
+                    return header;
+                });
+            };
+
+            recording.request.headers = processHeaders(recording.request.headers);
+            recording.response.headers = processHeaders(recording.response.headers);
+        });
+    }
+
+    removeBootstrapRequests(): void {
+        this.polly?.server.any().intercept((req, _, interceptor) => {
```

Comment:
Not sure why we need this. We either record or replay, there shouldn't be a third state where we record some endpoints but not others.

Reply:
Removed. 
As we only record a HAR after setting up environment, all recordings will be replayed now.